api/v1alpha2/sysprep/sysprep.go

```diff
@@ -67,7 +67,8 @@ type GUIUnattended struct {
 	// you may want to increase it. This number may be determined by the list of
 	// commands executed by the GuiRunOnce command.
 	//
-	// Please note this field only matters if AutoLogon is true.
+	// Please note this field only matters if AutoLogon is true. AutoLogonCount
+	// must be specified if AutoLogon is true.
```

Comment:
This seems a little redundant. Do you think this is still clear if we remove the existing `Please note this field only matters if AutoLogon is true`?

We can add a webhook validation for this?

Reply:
yeah. how about ?
// Please note this must be specified if AutoLogon is true.

Reply:
I'll add some validation for this. good call.

Reply:
> yeah. how about ? // Please note this field must be specified if AutoLogon is true.

I would make a mention that it must be positive since it isn't a pointer and specifying zero is no good.